packages/components/src/tag/docs/TagGroup.stories.tsx

```diff
@@ -0,0 +1,300 @@
+import { SparklesIcon } from "@hopper-ui/icons";
```

Comment:
I can see at least 3 more use cases:

- Customize the empty list: https://react-spectrum.adobe.com/react-aria/TagGroup.html#empty-state
- Tag as links: https://react-spectrum.adobe.com/react-aria/TagGroup.html#links
- Tag as router links:https://react-spectrum.adobe.com/react-aria/TagGroup.html#client-side-routing (we have examples in button")

Reply:
alright, I'll work on there, thanks